src/index.html

```diff
@@ -4,9 +4,236 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header_logo-image"  href="index.html"  ><img src="/images/logo.png" alt="logo"></a>
+      <nav class="header_nav">
+        <ul class="header_nav__list">
+          <li>
+            <a class="header_nav__link is-active" href="https://apple">Apple</a>
+          </li>
+
+          <li><a class="header_nav__link"  href="https://samsung">Samsung</a></li>
```

Comment:
Need to fix everywhere
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active" href="#apple">Apple</a>
          </li>

          <li class="nav__item">
            <a class="nav__link"  href="#samsung">Samsung</a>
          </li>
```